community/gsoc.md

```diff
@@ -0,0 +1,60 @@
+---
+layout: website-normal
+title: Google Summer of Code
+---
+
+<div class="panel panel-default">
+  <div class="panel-heading" markdown="1">
+#### Current Status
+  </div>
+  <div class="panel-body" markdown="1">
+The Apache Software Foundation is participating in 2018's Google Summer of Code!
+Apache Brooklyn, along with our fellow projects at the ASF, are welcoming
+contact from potential GSoC students. You are welcome to work on your own ideas
+for our project, or look at the [list of project ideas](http://s.apache.org/gsoc2018ideas)
+(note that this list is for *all* of the Foundation's projects, not just
+Brooklyn.)
+
+Applications open on **March 12th 2018** and must be received by **March 27th
+2018**. It will greatly benefit your application if you join our community and
+discuss your proposals with us before you make your application - see below for
+more details.
+  </div>
+</div>
+
+## Potential students
+
+Thank you for considering Apache Brooklyn and The Apache Software Foundation for
+your GSoC project. Welcome to our community!
+
+Firstly, if you have not already done so, please read the [Student Guide to
+Google Summer of Code](https://google.github.io/gsocguides/student/). It
+contains essential information about how to approach your project before and
+after your application is submitted. Also, please read the [GSoC information
+provided by The Apache Software Foundation](https://community.apache.org/gsoc.html),
+too.
+
+Secondly, Apache values community, and the community and all its work is done in
+the open, on public mailing lists and IRC channels. Please [subscribe to our
+mailing lists](https://brooklyn.apache.org/community/mailing-lists.html) and
```

Comment:
This you can use relative links for this?

Reply:
👍